scripts/inc.compatibility.sh

```diff
@@ -263,15 +263,13 @@ webviewstock"
 
 api23hack(){
   if [ "$API" -ge "23" ]; then
-    gappscore="$gappscore
-packageinstallergoogle"
     gappspico="$gappspico
 googletts"
     gappsmini="$gappsmini
 calculatorgoogle"
     gappsstock="$gappsstock
```

Comment:
I think it might be in general a good idea to treat PackageInstaller as we do with calsync, since I consider it quite a "core" app. Would you find it a good idea to add it to "gappspico" instead? It will then still be keyword configurable, but included for 99% of the users.